### PR TITLE
Documentation of steps for machine learning

### PR DESCRIPTION
**Pull request by reenz**
:shipit:

### CODE REVIEW

**Review by domvernon — APPROVED** [review]
:shipit:

**Review by ealitten — APPROVED** [review]

File: pokemon/documentation.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+import pandas as pd
+
+## Step 1- Data Munging
+
+# Storing ‘combats.csv(win/lose data)’ file data into a dataframe variable data
+data = pd.read_csv('./pokemon-challenge/combats.csv')
+
+# To replace winner/loser pokemon id with 0 and 1, loop over data rows-
```
Comment (Gleoman) [inline pokemon/documentation.md]: Minor consistency point - perhaps unnecessary "-" at end of this line.  Recommend change this or else bring every other sub-heading into line with it.

File: pokemon/documentation.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+import pandas as pd
+
+## Step 1- Data Munging
```
Comment (Gleoman) [inline pokemon/documentation.md]: Perhaps have top heading larger than sub-headings?

**Review by Gleoman — APPROVED** [review]
Couple of minor comments, otherwise good to go :shipit:

**Pull request merged**